### ISSUE TITLE
Autocompletion like nodejs

### ISSUE THREAD

**Issue by joshuarossi** [jhmaster2000/bun-repl#6]
Would be nice to have, this is just a feature request.

**Comment by jhmaster2000** [comment]
Planned soon. This is just currently blocked by a bug in bun's readline implementation, will be done as soon as that's fixed.

**Comment by joshuarossi** [comment]
awesome! makes it so much more helpful to explore what different things are available

**Comment by CreativeBuilds** [comment]
has there been an update on this @jhmaster2000 ? tab complete would be 10/10 rn
